docs/guides/algorithmiq-tem.ipynb

```diff
@@ -153,7 +153,11 @@
     "\n",
     "tem_function_name = \"algorithmiq/tem\"\n",
     "\n",
-    "catalog = QiskitFunctionsCatalog()\n",
+    "catalog = QiskitFunctionsCatalog(\n",
+    "    channel=\"ibm_cloud\",\n",
+    "    instance=\"USER_CRN\",\n",
+    "    token=\"USER_API_KEY\"\n",
+    ")\n",
```

Comment:
Per @pandasa123, the new code for the authentication step should be 

`catalog = QiskitFunctionsCatalog(
    channel="ibm_cloud / ibm_quantum / ibm_quantum_platform",
    instance="USER_CRN / HGP",
    token="USER_API_KEY / IQP_API_TOKEN"
)`

(apologies that I can't make this change myself; I don't have access to write to this branch, even after trying to fork, clone, etc.)

Reply:
To be clear - the formatting should look like this:
<img width="598" alt="Screenshot 2025-06-06 at 2 31 43 PM" src="https://github.com/user-attachments/assets/91cbdab5-a9fc-4faa-b3f5-1b3b5c9a3712" />